99-ocf.pm

```diff
@@ -14,6 +14,24 @@ Set($WebFallbackToRTLogin, 1);
 Set($WebRemoteUserAutocreate, 1);
 Set($WebRemoteUserGecos, undef);
 
+Set($ExternalInfoPriority, ['ldap']);
+# Fetch EmailAddress from the ocfEmail LDAP attribute
+Set($ExternalSettings, {
+	'LDAP' => {
+		'type'             =>  'ldap',
+		'server'           =>  'ldap.ocf.berkeley.edu',
+		'base'             =>  'ou=People,dc=OCF,dc=Berkeley,dc=EDU',
+		'filter'           =>  '(objectClass=ocfAccount)',
+		'attr_match_list'  => [
+			'Name',
+		],
+		'attr_map' => {
```

Comment:
Nit: what is the deal with this unique style of arrow formatting

Reply:
Copied from the docs: https://docs.bestpractical.com/rt/4.4.4/RT/Authen/ExternalAuth.html